web/src/pages/HomePage/HomePage.jsx

```diff
@@ -232,6 +240,14 @@ const HomePage = () => {
 
       setTasks(orderedTasks);
     }).catch(() => {
+      const convertedDate = new Date(date);
+      const formattedDate = convertedDate.toISOString();
+      const inputTask = {
+        date: formattedDate,
+        taskList: EMPTY_TASKS_DATA,
+        userId: currentUser.id,
+      };
+      create({variables: {input: inputTask}})
```

Comment:
If I'm reading this code correctly, I don't know if we want to be creating empty tasks lists in the database. In my opinion, this code belongs as a check when trying to *update* the database. If there wasn't already something there, then we need to create. But an empty task list should never be created, hence the check on line 193.

![image](https://github.com/R2bEEaton/njit-pending-cs490/assets/34921506/238f6bf5-4064-4bd1-866d-567bd2d51895)

Reply:
Per our conversation, merging